src/actions/index.js

```diff
@@ -78,12 +78,27 @@ export function fetchSectionComments(hearingId, sectionId) {
   };
 }
 
-export function postSectionComment(hearingId, sectionId, text, authorName, pluginData = null, authCode = "") {
+export function postSectionComment(hearingId,
```

Comment:
Okay, this is starting to have way, way too many arguments, and since JavaScript doesn't have named args, miscalls will be approximately 110% probable.

Maybe turn the signature into `hearingId, sectionId, commentData={}`, and validate the contents of the `commentData` within the function?